target/linux/realtek/rtl839x/config-5.10

```diff
@@ -137,6 +137,7 @@ CONFIG_MTD_SPLIT_EVA_FW=y
 CONFIG_MTD_SPLIT_FIRMWARE=y
 CONFIG_MTD_SPLIT_TPLINK_FW=y
 CONFIG_MTD_SPLIT_UIMAGE_FW=y
+CONFIG_MTD_VIRT_CONCAT=y
```

Comment:
Please put this change in a separate commit. You're adding the first device that will use it, so that could the be rationale you can put in the commit message.